concepts/staking.md

```diff
@@ -6,40 +6,42 @@ description: Details of Staking with Angle Protocol
-
-400,000,000 ANGLE (40% of supply) are planned to be distributed through staking. The amount being distributed are divided by 1.5^(1/52) = 1.007827 every week, equivalent to dividing the distribution by 1.50 every year.
+400,000,000 ANGLE (40% of the total supply) is being distributed through staking. The amount being distributed are divided by 1.5^(1/52) = 1.007827 every week, equivalent to dividing the distribution by 1.50 every year.
 
 ![ANGLE Distribution](../.gitbook/assets/Liquidity-Mining-Distributed-Supply-Over-Time.png)
 
 We built some simulations to evaluate how supply evolves over time, you can take a look at this [Google Sheet](https://docs.google.com/spreadsheets/d/1yraSUH_7D-VMnCUsIYWWdW1pxL7bDxN3o0M5japQmeY/edit#gid=0).
 
+## 🪐 Gauges and rewards allocation
+
+Each staking contract receiving rewards should have a dedicated gauge. Gauges are used so that veANGLE holders can dedicate their voting powers to influence the distribution of ANGLE rewards between the different staking contracts.
+
+veANGLE holders assign specific weights of their voting power to the different gauges, and the sum of all the veANGLE assigned to each gauge by all holders will determine the quantity of rewards to be distributed. Once weights are allocated, they will be reverberated in the following weeks without users having to do anything except if they want to change it.
+
 {% hint style="info" %}
-ANGLE token owners are able to participate to Snapshot votes each week to decide how to allocate the liquidity mining supply across different pools. This is detailed [here](../governance/voting.md)
+The rewards weight allocation can change each week depending on the changes in voting power allocated. The process is detailed [here](../governance/veANGLE/gauges.md).
 {% endhint %}
 
-## 📶 Token Accumulation
+## 💥 Boost on rewards
+
+Stakers on contracts internal to Angle can boost the rewards they receive by holding veANGLE. Note that this doesn't impact the inflation rate, and only change the rewards they receive compared to other LPs on this pool.
 
-At each block, tokens are distributed based on the rate specified by governance. If 1000 governance tokens are to be distributed for in 100 blocks (e.g, 10 governance tokens distributed each block), then stakers receive at a each block a fraction of these 10 tokens depending on what they put in the staking contract and on what other stakers have put at the time of the block.
+This boost can go up to x2.5 the base quantity of rewards, and depends on the liquidity on the staking contract and the veANGLE balance of the stakers. All the information about the boost can be found on the [boost](../governance/veANGLE/boost.md) page.
```

Comment:
Here, we should add rewards about perps and boosts do not apply

Reply:
already in the info below